tests/wpt/css-tests/selectors-3_dev/xhtml1/css3-selectors-lang-014.xht

```diff
@@ -27,7 +27,7 @@ This tests a detail related to :lang support. If :lang is not supported, a messa
 <script>
 test(function() {
 assert_equals(document.getElementById('colonlangcontroltest').offsetWidth, 0)
-assert_equals(document.getElementById('box').offsetWidth, 50);
+assert_equals(document.getElementById('box').offsetWidth, 100);
```

Comment:
as the spec says, 
``` 
For example, the extended language range "de-*-DE" (or its synonym "de-DE") matches all of the following tags:
 de-de (German, as used in Germany)
de-Latn-DE (Latin script)
```
so it is the same test case over, which should succeed. But am not too sure on this since browsers implemented it differently.